3-async-await/solution.js

```diff
@@ -19,17 +19,50 @@ Example:
 8. add any needed adjustment to solution() function
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
-
-function solution() {
-    // YOUR SOLUTION GOES HERE
-
+const getPrices = require('./prices');
+const getProduct = require('./products');
+async function solution() {
+  try {
     // You generate your id value here
-
+    const id = generateId();
     // You use Promise.all() here
-
-    // You use Promise.allSettled() here
-
-    // Log the results, or errors, here
+    const [price, name] = await Promise.all([getPrices(id), getProduct(id)]);
+    const product = { id, product: name, price };
+    console.log(`\nResolving with Promise.all:\n`, product);
```

Comment:
I don't understand this fully assignation.
Is name a string? if so then as an advice, you can use more descriptive variables, like productName or something like that.